docs/source/yelpsoa_configs.rst

```diff
@@ -236,6 +236,14 @@ For more information on selector operators, see the official Kubernetes
 documentation on `node affinities
 <https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#node-affinity>`_.
 
+  * ``pod_management_policy``: An option for applications managed with `StatefulSets <https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/>`_ to determine if the pods are managed in parallel or in order.
```

Comment:
you'll also need to update the schema for yelpsoa-configs here https://github.com/Yelp/paasta/blob/master/paasta_tools/cli/schemas/kubernetes_schema.json

Reply:
thanks, just added 😄